packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -512,14 +514,11 @@ class SystemChrome {
   /// If a particular style is not supported on the platform, selecting it will
   /// have no effect.
   ///
-  /// {@tool snippet}
-  /// ```dart
-  /// @override
-  /// Widget build(BuildContext context) {
-  ///   SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle.dark);
-  ///   return const Placeholder();
-  /// }
-  /// ```
+  /// {@tool sample}
```

Comment:
In both of these, do you feel it is worth mentioning whether one will override the other? If a developer uses both with different colors for example, what happens to the status bar?

Reply:
It's a good point but there are several possible combinations (depending on where the AnnotatedRegion is placed). Existing AppBar.setSystemUIOverlayStyle documentation avoid this question by stating: "Apps should not enclose an AppBar with their own [AnnotatedRegion]" (this is above the see also section I updated).